android/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest
     package="i2p.bote.android"
-    xmlns:android="http://schemas.android.com/apk/res/android">
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:sharedUserId="net.i2p">
```

Comment:
Not sure why this is required, but I don't believe it will harm the app, and we can always remove it later if it is unnecessary.